docs/pages/api/events.md

```diff
@@ -0,0 +1,87 @@
+---
+description: WebSocket events
+---
+
+# Events
+
+LiveCompositor is using WebSocket connection to send events to the connected clients. Supported events are listed below.
+
+### `VIDEO_INPUT_DELIVERED`
+
+```typescript
+type Event = {
+  type: "VIDEO_INPUT_DELIVERED";
+  input_id: string;
+}
+```
+
+The compositor received the input, and the first frames of that input are ready to be used. If you want to ensure that some inputs are ready before you send the [`start`](./routes.md#start) request, you can wait for those events for specific inputs.
```

Comment:
I'm unsure if you should use "the first frames of that input are ready to be used" here. "ready" can be understood as: "it will be used for rendering". Without the "inside" knowledge of how compositor works, what queue does, etc. I would interpret it this way.

Consider pointing out that this doesn't guarantee that those input frames could be used in the next render and recommend using `VIDEO_INPUT_PLAYING`.

Reply:
> "ready" can be understood as: "it will be used for rendering"

- If compositor is started and there is no offset then this understanding is correct.
- If compositor was not started then distinction between those 2 events is obvious
- If input stream has offset defined then it's also 

> Consider pointing out that this doesn't guarantee that those input frames could be used in the next render and recommend using VIDEO_INPUT_PLAYING.

It does gurantee it if there is no offset.

Reply:
Ok, I understood it wrong the first time. I thought that this event was sent as soon as the first frame was sent to the queue, but now I see, that it's emitted on the buffering stage change.